app/controllers/agents/events_controller.rb

```diff
@@ -3,7 +3,8 @@ class EventsController < ApplicationController
     before_action :load_agent
 
     def index
-      @events = @agent.events.order(created_at: :desc)
+      @events = @agent.events.where.not(type: 'Scanned').order(created_at: :desc)
```

Comment:
Line is too long. [81/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.